sdk-tests/src/test/java/io/dapr/it/BaseIT.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.it;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.logging.LogManager;
+
+public abstract class BaseIT {
```

Comment:
It may be better to rename this so "IT"  is not the suffix so that it doesn't think this is a test based on the file name (e.g BaseIntegrationTest).

Reply:
BaseIntegrationTest is also IT because of the suffix. Still, I am renaming it.